java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/jdbc/PreparedStatement.java

```diff
@@ -272,13 +272,12 @@ protected void dataBuild() throws SQLException {
                 columnTypes.AddColumnType(types.get(i + 1));
             }
             this.currentRow = SQLRequestRow.CreateSQLRequestRowFromColumnTypes(columnTypes);
-            // TODO(hw): check if null
+            if (this.currentRow == null) {
+        	throw new SQLException("failed to build data with row");
```

Comment:
exception message should be `fail to create sql request row from column types`

Reply:
ok editing it